### PR TITLE
fix typo: excpetion -> exception

### PR DESCRIPTION
**Pull request by laszlovandenhoek**
This typo bothers me more than it should, but I see it in a lot of stack traces, where it grabs my attention even though it has nothing to do with the problem at hand.

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

File: akka-actor/src/main/scala/akka/pattern/AskSupport.scala [hunk 1]
```diff
@@ -284,11 +284,21 @@ object AskableActorRef {
...
 
+  /**
+    * INTERNAL API
+    *
+    * @deprecated
+    */
+  @InternalApi
+  @deprecated("use recipientTerminatedException instead") private[akka] def recipientTerminatedExcpetion(recipient: Any, message: Any, sender: ActorRef): AskTimeoutException = {
+    recipientTerminatedException(recipient, message, sender)
+  }
```
Comment (johanandren) [inline akka-actor/src/main/scala/akka/pattern/AskSupport.scala]: That's internal API so no need to keep the old name.

**Comment by johanandren** [comment]
Test failure is formatting, if you compile after change it will auto-format the required way.

File: akka-actor/src/main/scala/akka/pattern/AskSupport.scala (hunk 1 above)
Comment (laszlovandenhoek) [reply akka-actor/src/main/scala/akka/pattern/AskSupport.scala]: Fixed by adding a mima filter.

**Comment by laszlovandenhoek** [comment]
Now, 2.11 went OK but the 2.12 half of the build failed in Travis due to to this:

```
$ curl -sL https://raw.githubusercontent.com/shyiko/jabba/0.11.1/install.sh | bash && . ~/.jabba/jabba.sh
Installing v0.11.2...
/home/travis/.jabba/bin/jabba does not appear to be a valid binary.
Check your Internet connection / proxy settings and try again.
If the problem persists - please create a ticket at https://github.com/shyiko/jabba/issues.
The command "curl -sL https://raw.githubusercontent.com/shyiko/jabba/0.11.1/install.sh | bash && . ~/.jabba/jabba.sh" failed and exited with 1 during .
```

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Review by chbatey — APPROVED** [review]
LGTM

**Pull request merged**